### PR TITLE
Finalize old blobs

### PR DESCRIPTION
**Pull request by idruzhitskiy**

### CODE REVIEW

File: src/api/http.rs [hunk 1]
```diff
@@ -163,6 +164,17 @@ fn vdisks(bob: State<BobServer>) -> Json<Vec<VDisk>> {
     Json(vdisks)
 }
 
+#[delete("/blobs/outdated")]
+fn finalize_outdated_blobs(bob: State<BobServer>) -> Result<StatusExt, StatusExt> {
+    let backend = bob.grinder().backend();
+    runtime().block_on(backend.cleanup_outdated(1));
```
Comment (agend) [inline src/api/http.rs:170]: that could take a while - so don't block

File: openapi.yaml [hunk 2]
```diff
@@ -142,6 +142,15 @@ paths:
                 type: string
         default:
           $ref: '#/components/responses/DefaultError'
+  /blobs/outdated:
+    delete:
+      summary: Finalize old blobs
```
Comment (agend) [inline openapi.yaml:147]: I would extend: Close active blobs not recently used

File: src/core/backend/pearl/holder.rs [hunk 3]
```diff
@@ -58,6 +64,20 @@ impl Holder {
         self.start_timestamp <= timestamp && timestamp < self.end_timestamp
     }
 
+    pub(crate) fn is_outdated(&self) -> bool {
```
Comment (agend) [inline src/core/backend/pearl/holder.rs:67]: I think it's better to close not recently used blobs. It should be easy to track last time blob was written.

File: src/core/backend/core.rs [hunk 4]
```diff
@@ -296,4 +296,24 @@ impl Backend {
             None
         }
     }
+
+    pub(crate) async fn cleanup_outdated(&self, max: usize) {
```
Comment (agend) [inline src/core/backend/core.rs:300]: cleanup or close?

File: src/core/backend/pearl/holder.rs (hunk 3 above)
Comment (agend) [reply src/core/backend/pearl/holder.rs:67]: As well we can set not to close too small blobs

File: src/core/backend/core.rs (hunk 4 above)
Comment (idruzhitskiy) [reply src/core/backend/core.rs]: close, typo

File: src/api/http.rs (hunk 1 above)
Comment (pyakushin) [reply src/api/http.rs]: it's different runtime, it won't block main RT

**Pull request merged**